recipes/mkdocs-marimo/recipe.yaml

```diff
@@ -0,0 +1,45 @@
+context:
+  name: mkdocs-marimo
+  version: 0.1.2
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+- url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/mkdocs_marimo-${{ version }}.tar.gz
+  sha256: 9ac2fb08850f2f0f5ab01d06673b18425e678be660d5493ab7e7810993e60fce
+
+build:
+  script: python -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

Reply:
i thought in v1 recipes we don't use `{{ PYTHON }}` anymore? https://github.com/conda-forge/staged-recipes/blob/a8e5dbc12b38de1a6472d983399ac88c1ebfcd99/recipes/example-v1/recipe.yaml#L50

Reply:
Not sure. We still don't have docs on that for reviewers. I believe the extra options there are still welcomed though.